src/views/newConnection.tsx

```diff
@@ -1,42 +1,118 @@
 import Html from '@kitajs/html'
 import { singleton } from 'tsyringe'
 import { CompanyProfile } from '../models/companyHouseEntity'
-import { Page } from './common'
+import { FormButtonIcon, Page } from './common'
+
+export type CompanyProfileText =
+  | {
+      status: 'success'
+      company: CompanyProfile
+    }
+  | {
+      status: 'error'
+      errorMessage: string
+    }
+
+export type FormStage = 'form' | 'confirmation' | 'success'
 
 @singleton()
 export default class newConnectionTemplates {
   constructor() {}
 
-  public formPage = ({ targetBox }: { targetBox: JSX.Element }) => {
+  public formPage = (targetBox: CompanyProfileText, formStage: FormStage) => {
     return (
       <Page title="Veritable - New Connection" heading="New Connection" url="/connection/new">
-        <this.companyNumberInput targetBox={targetBox} />
+        <div class="connections header">
+          <span>Invite New Connection</span>
+        </div>
+        <this.companyFormInput targetBox={targetBox} formStage={formStage} />
       </Page>
     )
   }
 
-  public companyNumberInput = ({ targetBox }: { targetBox: JSX.Element }): JSX.Element => {
+  public companyFormInput = (params: {
+    targetBox: CompanyProfileText
+    formStage: FormStage
+    email?: string
+    companyNumber?: string
```

Comment:
we need to be more careful with these values as they come form the user. Can we please make sure they are escaped to avoid xss